### PR TITLE
9.3.0-SNAPSHOT

### PR DESCRIPTION
**Pull request by tristantarrant**

### CODE REVIEW

**Comment by ryanemerson** [comment]
@tristantarrant `InfinispanExtensionCliIT.testCliContainerCreation` is related:

```
{"outcome" => "failed","failure-description" => "WFLYCTL0310: Extension module org.infinispan.extension:ispn-9.2 not found","rolled-back" => true} expected:<[success]> but was:<[failed]>

```

**Comment by tristantarrant** [comment]
Fixed by using Version.moduleSlotI() in place of 'ispn-9.2'

**Comment by ryanemerson** [comment]
@tristantarrant Thanks, I definitely should have done that in the first place.

**Comment by ryanemerson** [comment]
Thanks @tristantarrant, 9.2.x branch created: https://github.com/infinispan/infinispan/tree/9.2.x

File: all/cli/pom.xml [hunk 1]
```diff
@@ -5,7 +5,7 @@
    <parent>
       <groupId>org.infinispan</groupId>
       <artifactId>infinispan-all-parent</artifactId>
-      <version>9.2.2-SNAPSHOT</version>
+      <version>9.3.0-SNAHPSHOT</version>
```
Comment (rvansa) [inline all/cli/pom.xml]: @tristantarrant Fat fingers, too much gnocco fritto!

Comment (ryanemerson) [reply all/cli/pom.xml]: Damn! I'm using the fact it's Monday as my excuse for also missing this

Comment (ryanemerson) [reply all/cli/pom.xml]: @tristantarrant Can you issue another PR using your new script please

Comment (tristantarrant) [reply all/cli/pom.xml]: I just pushed a fix

**Pull request merged**